lib/postgres_to_redshift/update_tables.rb

```diff
@@ -53,6 +61,11 @@ def tables
       Tables.new(source_connection: source_connection, target_connection: target_connection).all
     end
 
+    def keys
+      # do not cache - we want fresh key listing for retries
+      Keys.new(source_connection: source_connection).all
```

Comment:
Could we pass `tables` as an argument rather than reading from `REDSHIFT_INCLUDE_TABLES` in `keys.rb`?

Reply:
Sure!